src/game.rs

```diff
@@ -364,6 +375,7 @@ impl Game {
                     .get_mut_heister_from_vec(heister_color.clone())
                     .unwrap();
                 heister.map_position = dest_pos;
+                self.update_possible_placements();
```

Comment:
Like discussed offline, just do this in `get_game_state`.

Reply:
Not gonna do it there, because then it means we have to have mut ref to game//game_state in functions designed psecfiically to not mutate.

Calling once instead in self.process_message or whatever it's called, same idea of handling updates in only one place.